config/routes.rb

```diff
@@ -1,5 +1,5 @@
 if ShopifyApp.configuration.routes_enabled?
-  ShopifyApp::Engine.routes.draw do
+  Rails.application.routes.draw do
```

Comment:
so this will automatically generate the routes for any app that adds `shopify_app` gem?